### PR TITLE
Cvsl 1158 update approver view case list page

### PR DESCRIPTION
**Pull request by justin-gardiner**
Screenshot 2023-07-17 at 11 57 27

### CODE REVIEW

File: server/routes/changeLocation/handlers/changeLocation.ts [hunk 1]
```diff
@@ -11,7 +11,13 @@ export default class ChangeLocationRoutes {
...
       const caseload = prisonCaseloadFromNomis.map(c => ({ value: c.caseLoadId, text: c.description }))
       const checked = req.session.caseloadsSelected
-      const cancelToUrl = req.query.view ? '/licence/view/cases?view=probation' : '/licence/view/cases'
+
+      let cancelToUrl
+      if (req.query.view) {
+        cancelToUrl = '/licence/view/cases?view=probation'
+      } else {
+        cancelToUrl = `/licence/view/cases${req.query.approval ? `?approval=${req.query.approval}` : ''}`
+      }
```
Comment (andrewrlee) [inline server/routes/changeLocation/handlers/changeLocation.ts:20]: are there no tests for this module?

Comment (justin-gardiner) [reply server/routes/changeLocation/handlers/changeLocation.ts:20]: yes there are, will add a test

Comment (justin-gardiner) [reply server/routes/changeLocation/handlers/changeLocation.ts:20]: test added

File: assets/sass/components/approval-cases.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.approval-due-date .approval-overdue {
+  border-left: 5px solid #d4351c;
+  font-weight: bold;
+  padding-left: 5px;
+}
+
+.approval-due-date .approval-overdue-message {
+  color: #0b0c0c;
+  font-size: 16px;
```
Comment (andrewrlee) [inline assets/sass/components/approval-cases.scss:17]: not suggesting change it for this but think there might be font size helpers too

**Pull request merged**